src/libaktualizr/uptane/managedsecondary.cc

```diff
@@ -104,6 +107,11 @@ bool ManagedSecondary::putRoot(const std::string &root, const bool director) {
 }
 
 bool ManagedSecondary::sendFirmware(const std::string &data) {
+  install_thread = std::thread(&ManagedSecondary::sendFirmware_, this, std::ref(data));
+  return true;
```

Comment:
Are we going to need some sort of locking here to make sure we don't try to launch two `install_thread` instances? This seems just a bit dangerous.